src/constraints/outage_constraints.jl

```diff
@@ -56,11 +56,22 @@ function add_outage_cost_constraints(m,p)
     @expression(m, ExpectedOutageCost,
         sum(m[:dvMaxOutageCost][s] * p.s.electric_utility.outage_probabilities[s] for s in p.s.electric_utility.scenarios)
     )
+   
+    if !isempty(setdiff(p.techs.elec, p.techs.segmented))
+        @constraint(m, [t in setdiff(p.techs.elec, p.techs.segmented)],
+            m[:binMGTechUsed][t] => {m[:dvMGTechUpgradeCost][t] >= p.s.financial.microgrid_upgrade_cost_fraction * p.third_party_factor *
+                                    p.cap_cost_slope[t] * m[:dvMGsize][t]}
+        )
+    end
 
-    @constraint(m, [t in p.techs.elec],
-        m[:binMGTechUsed][t] => {m[:dvMGTechUpgradeCost][t] >= p.s.financial.microgrid_upgrade_cost_fraction * p.third_party_factor *
-		                         p.cap_cost_slope[t] * m[:dvMGsize][t]}
-    )
+    if !isempty(p.techs.segmented)
```

Comment:
can there be non-elec segmented techs?

Reply:
Nope, I just checked and AbsorptionChiller and Boiler, the only non-elec techs which have a capital cost, do not have any attribute that can make them segmented (cost curve or incentive inputs).

Reply:
ok, lets just create an issue to make this robust to that possibility in case it happens later